go.mod

```diff
@@ -96,6 +96,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20200410145947-61e04a5be9a6
 	k8s.io/kubectl v0.18.6
 	k8s.io/kubernetes v1.14.0
+	k8s.io/metrics v0.18.6
```

Comment:
Why there is a new module imported? I didn't see any code change depended on this module.

Reply:
Me neither. But after I run make `ks-apiserver`, the go.mod & go.sum files are changed. Should we remove the dependency?